packages/react-native-editor/package.json

```diff
@@ -63,7 +63,7 @@
 		"react-native-get-random-values": "1.4.0",
 		"react-native-linear-gradient": "2.7.3",
 		"react-native-modal": "13.0.1",
-		"react-native-reanimated": "2.17.0",
+		"react-native-reanimated": "3.6.2",
```

Comment:
Much like Android, upgrading to Reanimated 3 is required to address errors from incompatible builds/dependencies.